docs/clients/brushless_drive.rst

```diff
@@ -105,121 +105,121 @@ Message Table
++--------+-------------------------------+----------------+-----------+----------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| 51     | regen_limiting                | get            | uint8     | bool     | When dynamic supply current limiting is enabled, this reports whether the drive voltage is currently being limited due to the regeneration current back through the supply. True means that the limiter is currently controlling the drive voltage because the regeneration supply current is at or above the limit, false means the regeneration current is low enough that the limiter has not activated. |
++--------+-------------------------------+----------------+-----------+----------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| 52     | regen_limit_adjust            | get            | float     | V        | Reports the adjustment voltage being applied to the dynamic supply current limited volts by the closed loop controller due to the regeneration limit. The adjustment adapts to improve the accuracy of the supply current limiting.                                                                                                                                                                         |
++--------+-------------------------------+----------------+-----------+----------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| 53     | motoring_limiting             | get            | uint8     | bool     | When dynamic supply current limiting is enabled, this reports whether the drive voltage is currently being limited due to the motoring supply current. True means that the limiter is currently controlling the drive voltage because the supply current is at or above the limit, false means the supply current is low enough that the limiter has not activated.                                         |
```

Comment:
can we come up with a better descriptor than "motoring." We use it a bunch of places, but it doesn't really mean anything outside of _Sister Christian_ (which is why we put it in quotes the first time).

Reply:
At this point, I think we are too far gone on the use of motoring. It does mean something to us, it means you are acting as a motor and not a generator. We've baked into the names of the parameters even. That whole terminology change is beyond the scope of this task.